### PR TITLE
IncludeFixer - fix and test more cases

### PR DESCRIPTION
**Pull request by SpacePossum**
Fix the spacing for these cases as well:
```php
<?php require$a;
```
```php
<?php require'foo.php';
```

```php
<?php require/**/'foo.php';
```

### CODE REVIEW

File: Symfony/CS/Tests/Fixer/Symfony/IncludeFixerTest.php [hunk 1]
```diff
@@ -28,121 +29,170 @@ public function testFix($expected, $input = null)
...
-                '<?php $foo = (false === include($zfLibraryPath."/Zend/Loader/StandardAutoloader.php")) {$foo = "asd";};',
-            ),
-            array(
-                '<?php require_once SOME_CONST . "file.php"; require Foo::Bar($baz);',
-                '<?php require_once( SOME_CONST . "file.php" ); require Foo::Bar($baz);',
-            ),
-        );
+        $template = '<?php %s';
+        $tests = array();
+        foreach (array('require', 'require_once', 'include') as $statement) {
```
Comment (keradus) [inline Symfony/CS/Tests/Fixer/Symfony/IncludeFixerTest.php]: why not include once?

Comment (SpacePossum) [inline Symfony/CS/Tests/Fixer/Symfony/IncludeFixerTest.php]: just forgotten ;)
 let me check what the tests do...

Comment (SpacePossum) [inline Symfony/CS/Tests/Fixer/Symfony/IncludeFixerTest.php]: updated, all green :)

**Comment by SpacePossum** [comment]
ping @keradus it seems the test configuration for PHP7 is broken https://travis-ci.org/FriendsOfPHP/PHP-CS-Fixer/jobs/95780602 ?

**Comment by SpacePossum** [comment]
or maybe I should just rebase :/
sorry for the ping

**Comment by keradus** [comment]
@SpacePossum https://github.com/travis-ci/travis-ci/issues/5267 ;)

**Comment by SpacePossum** [comment]
thanks @keradus for figuring it out : D

**Pull request closed without merging**